Deploy/UtilityScripts/CpqApiHelper.py

```diff
@@ -115,10 +116,79 @@ def deleteGlobalScript(self, id):
         )
         return response
 
+    # Custom Table APIS
+
+    def getAllCustomTables(self):
+        print("121")
+        headers = self.getCTHeaderJwt()
+        headers["Content-Type"] = "application/json"
+        headers["accept"] = "*/*"
+        url = self.__host + "api/custom-table/v1/customTables"
+        # tables = requests.get(url, headers=headers)
+        tables= self.testCallSuccess(
+            requests.get,
+            url,
+            headers=headers
+        )
+        print('131')
+
+        if tables.status_code == 200:
+            tables_data = tables.json()
+            return tables_data
+        else:
+            print(f"Request failed with status code: {tables.status_code}")
+
+    def updateCustomTables(self, table_name, payload):
+        data = payload
+        headers = self.getHeaderJwt()
+        headers["Content-Type"] = "application/json"
+        tableName = table_name
+        url = self.__host+ f"api/custom-table/v1/customTables/{tableName}"
+        tables= self.testCallSuccess(
+            requests.patch,
+            url,
+            headers=headers,
+            data=json.dumps(data)
+        )
+
+        print(tables)
+
+    def addCustomTables(self, mainData):
+        headers = self.getHeaderJwt()
+        headers["Content-Type"] = "application/json"
+        data = mainData
+        url = self.__host + "api/custom-table/v1/customTables"
+
+        response= self.testCallSuccess(
+            requests.post,
+            url,
+            headers=headers,
+            data=json.dumps(data)
+        )
+
+        if response.status_code == 201:
+            response_data = response.json()
+            print("Response:", response_data)
+        else:
+            print(f"Request failed with status code: {response.status_code}")
+
+    def deleteCustomTables(self, name):
+        headers = self.getHeaderJwt()
+        tableName=name
+        url = self.__host + f"api/custom-table/v1/customTables/{tableName}"
+        deleted_table= self.testCallSuccess(
+            requests.delete,
+            url,
+            headers=headers
+        )
+        print("deleted table " + str(deleted_table))
+
     # UTILITY METHODS
 
     def testCallSuccess(self, func, url, data='', params='', headers=''):
+        print("205" + str(url) + " " + str(data) + " " + str(params) + str(headers))
         response = func(url, data=data, params=params, headers=headers)
```

Comment:
Why 205, we are going to be making a ton of calls, let's not log these details unless we have an error.